README.md

```diff
@@ -1,12 +1,10 @@
 # News Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Get started with building responsive webpages, mobile first.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+This was all about "trial and error". 
```

Comment:
Would be nice to know more about your process and your thoughts on the project. What went well, what didn't work and what are you proud of? 😸

Reply:
Yes, I understand now why it can be good with a nice description :) I will update the README section for sure!